website/app/controllers/application_controller.rb

```diff
@@ -3,6 +3,8 @@
 class ApplicationController < ActionController::Base
   include CitizensAdviceComponents::Helpers
 
+  default_form_builder CitizensAdviceComponents::FormBuilder
```

Comment:
Does the documentation need to mention that `cads_button` is only available when the `CitizensAdviceComponents::FormBuilder` is used (I don't believe it's enabled by default in the design system gem)..?